lib/src/components/NxTag/NxTag.scss

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @import '../../scss-shared/nx-variables';
+ @import '../../scss-shared/nx-text-helpers';
+ @import '../../scss-shared/nx-container-helpers';
+
+.nx-tag {
+  @include semi-bold();
+  align-items: center;
+  background-color: $nx-blue-800;
+  border: 1px solid $nx-blue-800;
+  border-radius: 16px;
+  box-sizing: border-box;
+  color: $nx-text-color-dark;
+  cursor: default;
```

Comment:
Without this, we get the I bar cursor when hovered over the text.  Do we not want that?

Reply:
> Without this, we get the I bar cursor when hovered over the text. Do we not want that?

I assumed that we did not want the I-bar, I could see not value in selecting the text. @dturner32 ?

Reply:
correct, we don't need the |-bar

Reply:
> I could see not value in selecting the text.

Should we adjust `user-select` as well then?

Reply:
> > I could see not value in selecting the text.
> 
> Should we adjust `user-select` as well then?

https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/c59577e84d5ac3ad1ebc8761e2afb904c363546b